app/hooks/useActiveClaimCondition.tsx

```
@@ -0,0 +1,45 @@
+
+  useEffect(() => {
+    const getActiveCC = async () => {
+      try {
+        setIsLoading(true);
+        const res = await getActiveClaimCondition({
+          contract: props.contract,
+          tokenId: props.tokenId,
+        });
+
+        setActiveClaimCondition(res);
+      } catch (err: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)